test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -257,42 +257,40 @@ private static void getFailureReason() {
     }
 
     /**
-     * Position the instruction frame with testFrame ( testcase created
-     * frame) by the specified position
+     * Position the instruction frame with testWindow (testcase created
+     * window) by the specified position
      * Note: This method should be invoked from the method that creates
-     * testFrame
-     *
-     * @param testFrame test frame that the test is created
+     * testWindow
```

Comment:
It looks like you accidentally removed the empty line that served as the visual delimiter.

Reply:
@azvegint Thanks for catching it. Updated the PR with changes.